tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -175,6 +176,21 @@ sub run {
             $playbook_configs{spn_application_password} = get_var('AZURE_SPN_APP_PASSWORD', get_required_var('_SECRET_AZURE_SPN_APP_PASSWORD'));
         }
     }
+
+    my @addons = grep { defined $_ && $_ } split(/,/, get_var('SCC_ADDONS'));
+    # This implementation has a known limitation
+    # if SCC_ADDONS has two or more elements (like "ltss,ltss_es")
+    # only the last one will be added to the playbook argument.
+    foreach my $addon (@addons) {
```

Comment:
Scan all extensions in SCC_ADDONS and only care about ltss, calculate SCC Extension name and reg code for it and pack them in the right format expected by ltss argument of create_playbook_section_list